nxt_editor/integration/unreal/__init__.py

```diff
@@ -28,7 +28,7 @@ def launch_nxt_in_ue():
         __NXT_WINDOW.show()
         __NXT_WINDOW.raise_()
     else:
-        __NXT_WINDOW = nxt_editor.show_new_editor()
+        __NXT_WINDOW = nxt_editor.show_new_editor(start_rpc=False)
```

Comment:
The "correct" way to do this is: 
https://github.com/nxt-dev/nxt_editor/blob/b9bb55fa92e3aa6f9a3304c5aa525ed9a9190f31/nxt_editor/integration/maya/plug-ins/nxt_maya.py#L24
https://github.com/nxt-dev/nxt_editor/blob/b9bb55fa92e3aa6f9a3304c5aa525ed9a9190f31/nxt_editor/integration/maya/plug-ins/nxt_maya.py#L101

Reply:
Turns out I was already doing it the correct way, but inserted this kwarg into the launch because of a red-herring I saw in the log.